HW0/P5/P5.txt

```diff
@@ -1 +1,57 @@
 # Your answers here.
+1.  255 seconds (by whatever method - either by consecutively adding the next
+    bag to a running total, or pairwise, 128 + 64 + 32 + 16 + 8 + 4 + 2 + 1).
+
+2.  Yes - the 8 cashiers can split the bags evenly (so each has 32 bags, which
+    take 31 seconds to add), then four of them each add their total to the
+    total of one of the other four (1 additional second), then two of the
+    remaining four add the remaining two pairs (1 more second), then one
+    cashier adds these results (1 more second), for 34 seconds total.
+
+3.  8 seconds (in the first second, 128 cashiers each add a pair of bags;
+    the next second, 64 add pairs of those results; and so on, until one
+    remaining cashier adds the final total in the 8th second).
+
+4.  Log_2(N) seconds (because the number of sums is halved each second).
```

Comment:
For the infinite cashiers time, is it int(log_2(N)), since the seconds cashiers take are all in integers.